backend/README.md

```diff
@@ -30,3 +31,19 @@ npm start
 ```
 
 The port on which the server is listening is reported in the console output.
+
+## Environment variables
+Here's the list of the envirnment variables that could to be configured.
```

Comment:
1) "environment" instead of "envirnment".
2) "could be configured" instead of "could to be configured".
3) Personally, I'd add an empty line above this one.

Reply:
fixed, thanks!